src/lib/routes/edge-api/index.ts

```diff
@@ -78,27 +85,28 @@ export default class EdgeController extends Controller {
             middleware: [
                 this.openApiService.validPath({
                     tags: ['Edge'],
+                    summary: 'Receive metrics from Edge',
```

Comment:
When you're calling this endpoint you want to **send** metrics, right? 

```suggestion
                    summary: 'Send metrics from Edge',
```